src/views/GovernanceTaskList/TaskListCta.tsx

```diff
@@ -206,8 +206,22 @@ export const DecisionCta = ({ intake }: { intake: SystemIntakeForm }) => {
     );
   }
 
+  if (intake.status === 'NOT_IT_REQUEST') {
```

Comment:
This file/component is pretty well tested. Let's capture these cases too.

Reply:
I don't understand what you mean here.

Reply:
Write tests in `src/views/GovernanceTaskList/index.test.tsx` that test the changes you made.

Reply:
Added, I just did pretty simple tests for detecting the right buttons and the alert. Not sure if we want to. something more complex involving a business case, etc.